modules/mongodb/src/main/java/org/testcontainers/containers/MongoDBContainer.java

```diff
@@ -91,10 +153,20 @@ protected void containerIsStarted(InspectContainerResponse containerInfo, boolea
     }
 
     private String[] buildMongoEvalCommand(final String command) {
+        final String authOptions =
+            " -u " + username + " -p " + password + " --authenticationDatabase " + DEFAULT_AUTHENTICATION_DATABASE_NAME;
         return new String[] {
             "sh",
             "-c",
-            "mongosh mongo --eval \"" + command + "\"  || mongo --eval \"" + command + "\"",
+            "mongosh mongo" +
```

Comment:
Should be just mongosh in this case. It is not related to this change but just taking care of it

```suggestion
            "mongosh " +
```

Reply:
done